packages/components/src/icon-button/index.js

```diff
@@ -25,13 +25,15 @@ function IconButton( props, ref ) {
 		tooltip,
 		shortcut,
 		labelPosition,
+		fill,
```

Comment:
I don't think we need to add a prop here. We went down that route before and reverted it:

- https://github.com/WordPress/gutenberg/pull/14631
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/628

I think it should be possible to inject that prop into the icon `SVG` directly instead of passing it through the icon button chain

Reply:
Ok, I will try to refactor it. That was the simplest way to handle it